js/src/pages/price-benchmark/index.scss

```diff
@@ -4,7 +4,28 @@
 	padding: 0 $grid-unit-40;
 }
 
-.gla-price-benchmark__comparison-chart {
-	margin: $grid-unit-40 0;
+.gla-price-benchmark-tour {
+
+	.woocommerce-tour-kit-step__heading img {
+		display: block;
+		margin: 0 auto $grid-unit-20;
+	}
+
+	.woocommerce-tour-kit-step__description {
+		font-size: $gla-font-base;
+		line-height: $gla-line-height-medium;
+	}
+
+	.gla-price-benchmark__comparison-chart {
```

Comment:
@ankitrox I noticed one thing. `.gla-price-benchmark__comparison-chart` is not part of the tour, so this CSS will never be applied. Can you revert that change please? Thanks!